.github/workflows/verify.yml

```diff
@@ -0,0 +1,62 @@
+name: Verify
+
+on: [pull_request]
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        # node-version: [10.x, 12.x]
+        node-version: [12.x]
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }} on ${{ matrix.os }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - uses: ./
+
+      - name: Configure Package Manager
+        run: |
+          echo Configuring NPM_READ_TOKEN globally for .npmrc
+          npm config set '//registry.npmjs.org/:_authToken' ${{ secrets.NPM_READ_TOKEN }}
+          npm whoami
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        # Use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install Dependencies
+        run: |
+          yarn install --frozen-lockfile
+
+      - name: Lint
+        run: |
+          yarn lint
+
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v2
+
+      - name: Test
+        run: |
+          # yarn test --coverage
+          yarn test
+
+      - uses: codecov/codecov-action@v1
```

Comment:
When using this action before it was taking a long time to load since it is container based, so thats why I went with using the REST api directly through their bash script - did you experience that at all?

Reply:
Looks like they switched to JavaScript action at some point then – https://github.com/codecov/codecov-action.